scripts/zones/Port_Jeuno/npcs/Red_Ghost.lua

```diff
@@ -3,59 +3,53 @@
 --  NPC: Red Ghost
 -- Standard Info NPC
 -----------------------------------
-require("scripts/globals/quests");
-require("scripts/globals/pathfind");
+require("scripts/globals/quests")
+require("scripts/globals/pathfind")
 -----------------------------------
 
 local path =
 {
--96.823616, 0.001000, -3.722488,
--96.761887, 0.001000, -2.632236,
--96.698341, 0.001000, -1.490001,
--96.636963, 0.001000, -0.363672,
--96.508736, 0.001000, 2.080966,
--96.290009, 0.001000, 6.895948,
--96.262505, 0.001000, 7.935584,
--96.282127, 0.001000, 6.815756,
--96.569176, 0.001000, -7.781419,
--96.256729, 0.001000, 8.059505,
--96.568405, 0.001000, -7.745419,
--96.254066, 0.001000, 8.195477,
--96.567200, 0.001000, -7.685426
-};
+    -97, 0, -4,
+    -96, 0, 8,
+    -97, 0, -8,
+}
+
 function onSpawn(npc)
-    npc:initNpcAi();
-    npc:setPos(dsp.path.first(path));
-    onPath(npc);
-end;
+    npc:initNpcAi()
+    npc:setPos(dsp.path.first(path))
+    onPath(npc)
+end
 
 function onPath(npc)
-    dsp.path.patrol(npc, path);
-end;
+    dsp.path.patrol(npc, path)
+    if npc:isFollowingPath() == false then
+        npc:pathThrough(dsp.path.first(path), PATHFLAG_WALK)
+    end
+end
 
 function onTrade(player,npc,trade)
-end;
+end
 
 function onTrigger(player,npc)
     local WildcatJeuno = player:getVar("WildcatJeuno");
     if (player:getQuestStatus(JEUNO,dsp.quest.id.jeuno.LURE_OF_THE_WILDCAT) == QUEST_ACCEPTED and player:getMaskBit(WildcatJeuno,15) == false) then
-        player:startEvent(314);
+        player:startEvent(314)
     else
-        player:startEvent(34);
+        player:startEvent(34)
     end
 
     -- wait until event is over
-    npc:wait();
-end;
+    npc:wait()
+end
 
 function onEventUpdate(player,csid,option)
-end;
+end
 
 function onEventFinish(player,csid,option,npc)
     if (csid == 314) then
-        player:setMaskBit(player:getVar("WildcatJeuno"),"WildcatJeuno",15,true);
+        player:setMaskBit(player:getVar("WildcatJeuno"),"WildcatJeuno",15,true)
     end
 
-    npc:wait(0);
+    npc:wait(0)
```

Comment:
https://github.com/DarkstarProject/darkstar/issues/6250#issuecomment-544594726
take the zero out and tell me if he crashes.